astropy/visualization/lupton_rgb.py

```diff
@@ -155,20 +159,32 @@ def _convert_images_to_uint8(self, image_r, image_g, image_b):
         for c in image_rgb:
             c *= fac
             with np.errstate(invalid='ignore'):
-                c[c < 0] = 0                # individual bands can still be < 0, even if fac isn't
+                c[c < 0] = 0  # individual bands can still be < 0, even if fac isn't
 
         pixmax = self._uint8Max
-        r0, g0, b0 = image_rgb           # copies -- could work row by row to minimise memory usage
-
-        with np.errstate(invalid='ignore', divide='ignore'):  # n.b. np.where can't and doesn't short-circuit
+        (
+            r0,
+            g0,
+            b0,
+        ) = image_rgb  # copies -- could work row by row to minimise memory usage
+
+        with np.errstate(
+            invalid='ignore', divide='ignore'
+        ):  # n.b. np.where can't and doesn't short-circuit
```

Comment:
As above - no point splitting up the line just because of the comment

Reply:
Fixed by 5dded7a.